components/exception_handlers.py

```diff
@@ -0,0 +1,66 @@
+from fastapi import Request, status
+from fastapi.exceptions import RequestValidationError
+from fastapi.responses import JSONResponse
+from sqlalchemy.exc import IntegrityError
+
+from components.enums import JSENDStatus
+from components.exceptions import BackendException
+from settings import Settings
+from starlette.exceptions import HTTPException as StarletteHTTPException
+
+
+def backend_exception_handler(request: Request, exc: BackendException) -> JSONResponse:
+    """Return result from Back-end exception."""
+    response = JSONResponse(content=exc.dict())
+    response.status_code = exc.code
```

Comment:
You can use directly `status_code` inside initialization of JSONResponse.

Reply:
ok